test/e2e/PatchAppTest.hs

```diff
@@ -92,3 +93,21 @@ test_patch_accept = testCaseSteps "patch accept" $ \step -> do
 
     logOutput <- runTestCommand "git" ["log", "master"]
     assertContains logOutput "first commit"
+
+test_patch_unauthorized :: TestTree
+test_patch_unauthorized = testCaseSteps "patch unauthorized actions" $ \step -> do
+    step "init project with patch (maintainer)"
+    initProjectWithPatch
+    _ <- runTestCommand "git" ["checkout", "master"]
+
+    step "rad patch reject fails (contributer)"
+    _ <- runTestCommand "rad-key" ["create", "--force"]
+    rejectOutput <- runTestCommandForError "rad-patch" ["reject", "0"]
+    assertContains rejectOutput "Are you authorized to do so?"
```

Comment:
I think it would be less brittle to check for `"error"`. Unrelated to this PR, but I note that in `error-messages.rad`, some messages are prefixed with `error:`, some with `Error:` and some not prefixed at all. Maybe this should be made consistent.

Reply:
:+1: I can create another issue for that